subxt/src/lib.rs

```diff
@@ -54,7 +54,7 @@ use codec::{
     Encode,
 };
 use core::fmt::Debug;
-use derivative::Derivative;
+pub use derivative::Derivative;
```

Comment:
I'd lean towards just manually implementing clone instead of exposing Derivative in any way myself :)